src/formatters/index.js

```diff
@@ -46,8 +46,23 @@ export function generateStats(data: Array<CheckstyleItem>): Stats {
 }
 
 export function formatOutput(format: string, data: Array<CheckstyleItem>) {
-  if (!{}.hasOwnProperty.call(formatters, format)) {
-    throw new Error(`Formatter with name '${format}' does not exist.`)
+  let formatter
+  if (/^require:/.test(format)) {
```

Comment:
I was thinking it could try to load the external formatter if the name is not one of the builtin ones, instead of having to specify require, but maybe it's better to be explicit?